src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -332,6 +332,19 @@ public String getTopLevelIndexFileImportName() {
     return Joiner.on(File.separator).join(newNames.toArray());
   }
 
+  @Override
+  public String getCredentialsClassImportName() {
+    List<String> parts = getApiModules();
+    // Place credentials in top-level namespace.
+    parts = parts.subList(0, parts.size() - 1);
```

Comment:
Another instance of popping the last module that ideally should be consolidated into common logic.